README.md

```diff
@@ -97,7 +102,41 @@ Each lion should have:
 Create a new lion instance with the name `simba`
 
 ```ruby
-# code here
+class Animal
+  def initialize(name)
+    @name = name.capitalize
+  end
+
+  def greet
+    puts "Hello, my name is #{@name}"
+  end
+
+  def get_name()
+    @name
+  end
+
+  def set_name(name)
+    @name = name
+  end
+end
+
```

Comment:
this is correct, but remember the shortcut we learned in class? `attr_accessor`?